csympy/lib/csympy_wrapper.pyx

```diff
@@ -249,6 +249,31 @@ cdef class Symbol(Basic):
         import sympy
         return sympy.Symbol(str(deref(X).get_name().decode("utf-8")))
 
+
+def symbols(names, *args):
```

Comment:
I did so as we are not using assumptions currently. But on second thought, I agree with you.